server/src/main/java/com/cloud/network/NetworkModelImpl.java

```diff
@@ -1658,6 +1658,9 @@ public void checkNetworkPermissions(Account owner, Network network) {
             if (owner.getType() != Account.ACCOUNT_TYPE_PROJECT && networkOwner.getType() == Account.ACCOUNT_TYPE_PROJECT) {
                 User user = CallContext.current().getCallingUser();
                 Project project = projectDao.findByProjectAccountId(network.getAccountId());
+                if (project == null) {
```

Comment:
Same as previous comment

Reply:
here too, the check is explicitly for networks owned by a project